aleph-client/src/connections.rs

```diff
@@ -113,6 +114,18 @@ impl Connection {
 
         Ok(R::decode(&mut bytes.as_ref())?)
     }
+
+    pub async fn get_block_number_inner(
```

Comment:
the name suggests that it should be a private method (?)

Reply:
done